README.md

```diff
@@ -378,14 +378,30 @@ easier.
 You should not collect results intended for publication with development switches
 turned on.
 
-We also recommend that for 'real' benchmarking you turn on the
-`--hardware-reboots` and `--daemonise` switches. These ensure that the system
-will reboot before each benchmark execution, and that Krun will run in the
-background, allowing you to log out before the first reboot.
-
-You will also need to ensure that Krun is restarted once the machine has
-rebooted. The `etc/` directory contains example `/etc/rc.local` files for the
-platforms supported by Krun.
+We also recommend that benchmarking in "reboot mode". The recommended way to do
+this is via `cron(8)`. We supply a script `start_krun_from_cron` to make this
+easier.
+
+As a regular user (who has access to your experiment and the Krun code), invoke
+`crontab -e` and add lines similar to the following:
+
+```
+@reboot /path/to/krun/scripts/start_krun_from_cron /path/to/your/config.krun
+```
+
+The path to the config file must be an absolute path.
+
+If you wish to pass custom flags to Krun, you can append those onto the end of
+the invocation.
```

Comment:
How about something simpler "Options / flags after the `.krun` file will be passed as-is to Krun"?